aries-site/src/data/color.js

```diff
@@ -19,34 +19,49 @@ const primaryColors = [
 
 const coreColors = [
   {
-    name: 'purple!',
-    value: 'purple!',
-    hex: colors['purple!'],
+    name: 'neutral',
```

Comment:
Similar comment on this PR: https://github.com/grommet/hpe-design-system/pull/5412/files#r2445987654